hail/python/hail/table.py

```diff
@@ -1430,6 +1430,11 @@ def show(self, n=None, width=None, truncate=None, types=True, handler=None):
         handler : Callable[[str], Any]
             Handler function for data string.
         """
+        if n_rows is not None and n is not None:
```

Comment:
I'm a bit confused about this change

Reply:
it's also not typechecked which will create an error.

Reply:
I did modify the type check annotation.

In `Expression.show` I don't want to have an `if` that calls `show(n=n)` and `show(n=n_rows)`. I unified the two interfaces so that both `Table` and `MatrixTable` call the number of rows to be shown `n_rows` (and Table continues to support `n` for legacy reasons).

Reply:
oh, I see. OK, sure